templates/graphdb/statefulset.yaml

```diff
@@ -357,6 +451,60 @@ spec:
             - name: {{ .Values.import.volumeMount.volumeClaimTemplate.name }}
               mountPath: /opt/graphdb/home/graphdb-import
             {{- end }}
+            {{- with .Values.configuration.tls }}
+            {{- if .keystore.secretName}}
+            - name: graphdb-connector-keystore
+              mountPath: /etc/graphdb/connector-keystore.jks
+              subPath: {{ .keystore.keystoreKey }}
```

Comment:
The drawback of using subpaths is kubelet not being able to update the file when the secret is updated.